wiki/Visual_Settings/pl.md

```diff
@@ -0,0 +1,19 @@
+# Ustawienia wizualne
+
+![Nakładka ustawień wizualnych](./VisualSettings.jpg "Nakładka ustawień wizualnych")
+
+**Ustawienia wizualne** to ukryta nakładka, która znajduje się w dolnej części obszaru gry. Może zostać osiągnęty, gdy beatmapa jest ładowana. Rozpoczęcie beatmapy zostanie opóźnione, do czasu aż kursor opuści obszar nakładki. 
+
+Ewentualnie, nakładka ustawień wizualnych może zostać osiągnięta poprzez zapauzowanie gry. Jednakże, metoda ta nie zadziała w trakcie meczu w [trybie wieloosobowym](/wiki/Multi "Multi"), ponieważ zostanie to uznane jako próba wyjścia z meczu i opuszczenia pokoju.
```

Comment:
Nakładkę można także przywołać poprzez spauzowanie gry. Jednakże metoda ta nie zadziała w trakcie meczu w [trybie wieloosobowym](/wiki/Multi "Tryb wieloosobowy"), ponieważ zostanie to uznane jako próba wyjścia z meczu i gracz opuści pokój.